Shared/Samples/Change viewpoint/ChangeViewpointView.swift

```diff
@@ -0,0 +1,127 @@
+// Copyright 2022 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct ChangeViewpointView: View {
+    /// The view model for this sample.
+    @StateObject private var model = Model()
+    
+    @State private var visibleArea: ArcGIS.Polygon?
```

Comment:
```suggestion
    @State private var visibleArea: Polygon?
```
Can the prefix be removed?

Reply:
This is required for building to Mac Catalyst.